lib/repository/app_repository.dart

```diff
@@ -25,7 +25,7 @@ class AppRepository extends DemoDecoratorComposite {
 
   late final keysharesProvider = KeysharesProvider(_sdk.keygenState);
 
-  CancelableOperation<void> pair(QRMessage qrMessage, String userId, WalletBackup? backup) {
+  CancelableOperation<dynamic> pair(QRMessage qrMessage, String userId, WalletBackup? backup) {
```

Comment:
we should have a generic function that returns type based on use cases rather than having `dynamic` return here

Reply:
Actually return type is <void | pairing_data>, how can we do it in flutter?

Reply:
Previously, the function returned `void` to indicate that it won't return anything. So, now we want to return PairingData, we should have `PairingData?` as the returned type